catalog-entities/marketplace/plugins/analytics-provider-segment.yaml

```diff
@@ -84,6 +84,9 @@ spec:
           # highlight-start
           writeKey: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
           maskIP: true # prevents IP addresses from being sent if true
+          testMode: false # prevents data from being sent if true
+          appVersion: "1.0.0" # RHDH application version
```

Comment:
version 1.0?? the plugin is version 1.17 and the RHDH version is 1.8 -- is this being overwritten during a build process somewhere?

Reply:
well it's strange to set an example that doesn't reflect any sort of reality, but I guess YMMV.